x-pack/test/siem_cypress/es_archives/custom_rule_with_timeline/mappings.json

```diff
@@ -0,0 +1,7983 @@
+{
+  "type": "index",
```

Comment:
Are any of the other archives redundant with this one? If this is effectively `prebuilt_rules_loaded` + a timeline, I'm wondering whether the cost of maintaining both archives is worth the benefit.

Reply:
More generally: I'm wondering whether maintaining a superset archive of: rules, signals, index patterns, etc. would be more beneficial than several more specific archives.

Reply:
Not sure to be honest, I need to invest time on that one to have a unified set of archives, but that is better to do it in a different PR :)